packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -44,6 +42,10 @@ const StyledTab = styled(Button)`
 `
 
 const DEFAULT_HANDLER = () => true
+const DEFAULT_DATE_RANGE = {
+  endDate: '',
+  startDate: ''
+}
```

Comment:
Question about DEFAULT_DATE_RANGE throughout lib-user. In app-root page.js files, if `searchParams.end_date` and/or `searchParams.start_date` are undefined, the end date is defined as today and the start date if defined as 7 days ago by default. Is there a case where it makes sense for `endDate` or `startDate` to be empty strings in lib-user? If you're needing to check whether either variable exists in `getDateInverval()`, I suggest using `null` instead.

Reply:
Sounds good, updated with [1bedbf6](https://github.com/zooniverse/front-end-monorepo/pull/6160/commits/1bedbf6e92ee67c315cc07afa87daef6f3e69681)